lib/tasks/after_tasks.rake

```diff
@@ -500,6 +500,18 @@ namespace :After do
     end
   end
 
+  desc "Destroy wrangling assignments for non-canonical fandoms"
+  task(:destroy_noncanonical_wrangling_assignments => :environment) do
+    fandom_ids = Fandom.where(canonical: false).joins(:wrangling_assignments).pluck(:id)
+    assignments = WranglingAssignment.where(fandom_id: [fandom_ids])
```

Comment:
I suspect there is a better, one-step way to get the assignments for non-canonical fandoms, but I couldn't come up with it.

Reply:
I think `WranglingAssignment.joins(:fandom).where(tags: { canonical: false })` might work here.

Reply:
I think you are probably right! I was trying `WranglingAssignment.joins(:fandom).where(fandoms: { canonical: false })` and getting all sorts of fun messages about fandoms not existing, but it needing to be tags instead makes sense -- thanks!